lib/helpers.ts

```diff
@@ -24,21 +24,18 @@ export function getEditorCursorScopes(textEditor: TextEditor): Array<string> {
 }
 
 let minimatch: typeof import('minimatch')
-export function isPathIgnored(filePath: string | null | undefined, ignoredGlob: string, ignoredVCS: boolean): boolean {
+export async function isPathIgnored(
+  filePath: string | null | undefined,
+  ignoredGlob: string,
+  ignoredVCS: boolean,
+): Promise<boolean> {
   if (!filePath) {
     return true
   }
 
   if (ignoredVCS) {
-    let repository = null
-    const projectPaths = atom.project.getPaths()
-    for (let i = 0, { length } = projectPaths; i < length; ++i) {
-      const projectPath = projectPaths[i]
-      if (filePath.indexOf(projectPath) === 0) {
-        repository = atom.project.getRepositories()[i]
-        break
-      }
-    }
```

Comment:
Don't we need to check the project paths anymore?

Reply:
No, those cases are already covered by `atom.project.repositoryForDirectory`.